projects/cdk/utils/format/pad-start.ts

```diff
@@ -1,12 +1,10 @@
 /**
- * @deprecated: use {@link tuiPadStart} instead
+ * @deprecated use String.prototype.padStart in 3.0 (after Chrome 49 support is dropped)
  * Pads a string with symbols in the beginning
  *
  * @param sourceString
  * @param minResultLength
  * @param padString string to pad with
- *
- * @deprecated use String.prototype.padStart in 3.0 (after Chrome 49 support is dropped)
```

Comment:
Does eslint have the rule to place `@deprecated` at the top of js-doc ?

Reply:
Yes, if we will install https://github.com/gajus/eslint-plugin-jsdoc